### DISCUSSION TITLE
Should RYD show dislikes if the creator has disabled them?

### DISCUSSION THREAD

**Discussion by sy-b** [Anarios/return-youtube-dislike#530] [Polls]
RYD needs the number of likes to extrapolate the dislike count but if the creator disables both likes & dislikes of their video, then the extension shows "Dislike Disabled".
![unknown](https://user-images.githubusercontent.com/94835959/163172193-58648219-f176-4ecf-a348-87908d4ec062.png)
Reliable extrapolation is not possible on such videos.

There were requests to show votes by RYD users with a notification indicating that the vote count is solely by RYD users and isn't the real (like) & extrapolated (dislike) count,

**Continued in my comment below...**
(reason: votes are reset if I edit the post)

**Comment by sy-b** [comment]
The discussion thread of the discord server: https://discord.com/channels/909435648170160229/926874108308914256
(start reading from [here](https://discord.com/channels/909435648170160229/926874108308914256/926874110427033670))

---

Some points (from the discord thread) in favor & opposition of this (I'll add as more of these are mentioned here)

### Favor
i.e. Yes, show dislikes even if the creator has disabled likes & dislikes both.
- This is the point of this extension to "Return Dislikes"
- Creators who post Scams, fake tutorials, and clickbait often disable the vote count. Votes are meant to help people identify these.

<br>
 
- Creators shouldn't post if they can't handle the negativity.
- People not voting on a video isn't toxic and isn't a valid concern, people can be toxic in the comments of course (which the creator can delete)

<br>

### Opposition
i.e. No, don't show dislikes if the creator has disabled likes & dislikes both.
- We should respect creators' choices.
- This seemingly denies the creators their freedom to hide dislikes.
- This action might upset YouTube (which can slightly increase the chances of them acting against RYD)
- The creator might have a variety of reasons to disable stats
  - mental health
  - toxic community
  - or they just don't want to know what people think of their content

<br>

- Most of the votes here will be from viewers & not creators and hence are creators' opinions is underrepresented. (Any solution for this?)


## Note:

- Options 2, 3, 4, and 5 will be considered as "Yes".

**Comment by sy-b** [comment]
Any idea solve the creator underrepresentation problem?

**Comment by andry81** [comment]
You can add more options, like show both counters as a sum of [RYD]+[Google] counters, where [Google] is marked with the "approx" icon, like:

123<sup>[RYD]</sup> + 456<sup>[Google][approx]</sup>

**Reply by sy-b** [reply]
1. With 7th option, the comment fails to update. GitHub probably allows only 6 options.

2. What do you mean by "Google Counters"?

**Reply by andry81** [reply]
> 2\. What do you mean by "Google Counters"?

likes and dislikes counters

**Reply by sy-b** [reply]
But that (like counter) is what some creators are disabling & the question is about this exact thing.

Am I misinterpreting something?

**Reply by andry81** [reply]
Either the dislike counter or both counters can be disabled.

**Comment by cyrildtm** [comment]
I like writing options. Gimme ALL options in the world. I'll make the config page miles long.

**Reply by sy-b** [reply]
Please use one of these to get a list of all available options 
- `--help`
- `-h`
- `/?`
- `/h`

example usage: `$ homo-sapien-life --help world`

**Reply by cyrildtm** [reply]
@sy-b I tried but I only got `$: command not found`
I guess it's because I'm poor. I'll have to sell blood.

**Reply by sy-b** [reply]
Try using `$ snap install financial-literacy`

**Reply by cyrildtm** [reply]
Doesn't work either. You are really bad at it.

**Reply by sy-b** [reply]
![418](https://http.cat/418)
